src/Compilers/Core/Portable/Collections/BitVector.cs

```diff
@@ -283,18 +283,38 @@ public bool IntersectWith(BitVector other)
         /// <summary>
         /// Modify this bit vector by '|'ing each element with the other bit vector.
         /// </summary>
-        /// <param name="other"></param>
-        public void UnionWith(BitVector other)
+        /// <returns>
+        /// True if any bits were set as a result of the union.
+        /// </returns>
+        public bool UnionWith(BitVector other)
         {
+            bool anyChanged = false;
             int l = other._bits.Length;
+
             if (l > _bits.Length)
                 Array.Resize(ref _bits, l + 1);
```

Comment:
That's fine.  I'd clarify the comment though. 

---
In reply to: [75910660](https://github.com/dotnet/roslyn/pull/13234#discussion_r75910660) [](ancestors = 75910660)